### PR TITLE
The carriage return should not be in the regex

### PR DESCRIPTION
**Pull request by mvr320**
I'm not of the generation who has used a typewriter, so I forgot to take care of those.

In the future the whole file should be rewritten to be better understandable, I want to wait till this has proven itself.

Also implemented @RagnarGrootKoerkamp his idea, as its better than nothing

### CODE REVIEW

File: gitlab/visualcompare.sh [hunk 1]
```diff
@@ -64,8 +64,16 @@ do
         -X POST \
         -H "Authorization: token $GH_BOT_TOKEN_OBSCURED" \
         -H "Accept: application/vnd.github.v3+json" \
-        -d "{\"state\": \"$STATE\", \"target_url\": \"$CI_JOB_URL/artifacts/file/$FILE\", \"description\":\"UI changes\", \"context\": \"UI diffs ($dir)\"}"
-      exit 1
+        -d "{\"state\": \"$STATE\", \"target_url\": \"$CI_JOB_URL/artifacts/$FILE\", \"description\":\"UI changes\", \"context\": \"UI diffs ($dir)\"}"
+      if [ $STATE=="failure" ]; then
+        cat <<EOF > $failingchanges/README
+List you changed endpoints in a message on the PR. Regex is allowed.
```
Comment (nickygerritsen) [inline gitlab/visualcompare.sh:70]: This sentence feels odd?

Comment (mvr320) [reply gitlab/visualcompare.sh]: I think because it is, changed it a bit and made the syntax also slightly better

**Review by nickygerritsen — APPROVED** [review]

File: gitlab/visualgithubprdiscussion.py [hunk 2]
```diff
@@ -18,7 +18,7 @@ def toMessage(url):
     index=message.find(BEGIN_RGX)
     if index>-1:
         for line in message[index:].split('\n')[1:]:
```
Comment (thijskh) [inline gitlab/visualgithubprdiscussion.py:20]: Probably clearer to use `splitlines()` here and do away with the strip below?

Comment (mvr320) [reply gitlab/visualgithubprdiscussion.py]: Indeed better, is implemented now.

Locally you can see the difference now with:
```python3 gitlab/visualgithubprdiscussion.py public 910```
Where public is in the regex and is therefor a wanted change
```python3 gitlab/visualgithubprdiscussion.py anotherendpoint 910```
which is not in the regex and therefor would be a failing change

**Review by thijskh — APPROVED** [review]

**Pull request merged**